templates/sshd_config.erb

```diff
@@ -166,7 +166,9 @@ X11UseLocalhost <%= @sshd_x11_use_localhost %>
 PrintMotd <%= @sshd_config_print_motd %>
 #PrintLastLog yes
 #TCPKeepAlive yes
-TCPKeepAlive <%= @sshd_config_tcp_keepalive %>
+<% if @sshd_config_tcp_keepalive_real =~ /(yes|no)/ -%>
```

Comment:
Would `!= 'unset'` make this easier to change in the future as valid options to the parameter change?